src/index.ts

```diff
@@ -66,21 +74,25 @@ export type AcceptOffRampOrderParams = {
 
 export type RejectOrderParams = { orderId: string; reason: string };
 
-export type DataAccessParams = { userPK: string; secretKey: string };
+export type DataAccessParams = {
+  userPK: string;
+  secretKey: string;
+  includeValues?: boolean;
+};
 
 export type UserDataField = { dataId: string; status: ValidationStatus };
 
 export type UserDataValueField<T> = { value: T } & UserDataField;
 
 export type UserData = {
-  email: Array<UserDataValueField<string>>;
-  phone: Array<UserDataValueField<string>>;
-  name: Array<{ firstName: string; lastName: string } & UserDataField>;
-  birthDate: Array<UserDataValueField<Date>>;
-  document: Array<{ type: string; number: string; countryCode: string } & UserDataField>;
-  bankInfo: Array<{ bankName: string; accountNumber: string; bankCode: string } & UserDataField>;
-  selfie: Array<UserDataValueField<Uint8Array>>;
-  custom: Record<string, string>;
+  email?: UserDataValueField<string>;
```

Comment:
There still can be multiple values for each type, at least for document. For other types we can enforce single value.

Reply:
But actually, let's leave it like this for now. Probably, one document type would be enough.